app/mailers/user_mailer.rb

```diff
@@ -13,7 +13,7 @@ def contact(sender:, resource_id:, text:)
     @sender   = sender
     @receiver = User.find resource_id
     @text     = text
-    @subject  = I18n.t('email.user.contact.subject')
-    mail to: @receiver.email, subject: @subject
+    @subject  = I18n.t('email.user.contact.subject', sender_name: @sender.name)
```

Comment:
in the tests it is `sender.name`, while here we go for `sender.nickname`. We should use the same value.